src/DashboardService.Models.Dto/Requests/Group/PatchGroupRequest.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace LT.DigitalOffice.DashboardService.Models.Dto.Requests.Group;
+
+public record PatchGroupRequest
+{
+  public Guid BoardId { get; set; }
```

Comment:
can we change boardId of group by design? if yes - Guid? - make it nullable

Reply:
A task outside the board cannot exist